docs/src/modules.md

```diff
@@ -34,15 +34,40 @@ Other files needed by the module can also be stored within the
 
 ## Module template
 
-All modules should have an enable option created using `mkEnableTarget`.
+Modules should be created using `mkTarget` whenever possible.
+
+When using `mkTarget`, request it as a non-destructured arg before the optional
+module args. See the below example.
```

Comment:
Since you can't de-structure[^1] a function (unless it's a functor[^2]), it is redundant to mention de-structuring. Doing so in docs intended for new/external contributors will probably cause more confusion than clarity:

```suggestion
When using `mkTarget`, request it _before_ the optional module args.
See the below example.
```

[^1]: in the [functions section](https://nix.dev/manual/nix/2.28/language/syntax#functions) of the manual, they refer to de-structured args as a _"set pattern"_
[^2]: a functor is a callable attrset, see the [attrset section](https://nix.dev/manual/nix/2.28/language/syntax#attr-__functor) of the manual